### ISSUE TITLE
Truncated error output

### ISSUE THREAD

**Issue by wmdietlGC** [eisop/webserver#27]
Paste the following into the Live Demo:

```java
import org.checkerframework.checker.nullness.qual.Nullable;
import java.util.List;

class YourClassNameHere {
    <Z> void zap(List<@Nullable ? extends Z> r) {}
    
    void call(List<String> ls) {
        zap(ls);
    }
}
```

As output you get:

```
Error: [argument.type.incompatible] incompatible types in argument.
  found : @Initialized @NonNull List<@Initialized @NonNull String>
  required: @Initialized @NonNull List
```

instead of the full:

```
Demo.java:12: error: [argument.type.incompatible] incompatible types in argument.
                                            zap(ls);
                                                ^
  found   : @Initialized @NonNull List<@Initialized @NonNull String>
  required: @Initialized @NonNull List<? extends @Initialized @NonNull String>
1 error
```

Note how the error message is truncated at `<?`.

**Comment by CharlesZ-Chen** [comment]
Interestingly, browser would process "<? >" as html comment tag, and that is the reason why error output has been truncated...

I solved it by replacing "<?" to "< ?" in error outputs, to distinguish it with "<? >" comment tag.

**Comment by CharlesZ-Chen** [comment]
Fixed by PR #28
